ui/src/app/views/cohort-list/component.tsx

```diff
@@ -0,0 +1,147 @@
+import {Component} from '@angular/core';
+
+import {ResourceCard} from 'app/views/resource-card/component';
+
+import {
+  RecentResource,
+  WorkspaceAccessLevel
+} from 'generated/fetch';
+
+import {convertToResources, ResourceType} from 'app/utils/resourceActionsReact';
+
+import {cohortsApi} from 'app/services/swagger-fetch-clients';
+
+import {ClrIcon} from 'app/components/icons';
+
+import {TooltipTrigger} from 'app/components/popups';
+import {SpinnerOverlay} from 'app/components/spinners';
+
+import {CardButton} from 'app/components/buttons';
+import {navigate} from 'app/utils/navigation';
+import * as React from 'react';
+
+import {FadeBox} from 'app/components/containers';
+import {WorkspaceData} from 'app/resolvers/workspace';
+import {reactStyles, ReactWrapperBase, withCurrentWorkspace} from 'app/utils/index';
+
+const styles = reactStyles({
+  pageArea: {
+    display: 'flex',
+    justifyContent: 'flex-start'
+  },
+
+  cohortsHeader: {
+    color: '#2F2E7E',
+    fontSize: '20px',
+    lineHeight: '24px',
+    fontWeight: 600,
+    marginTop: 0,
+    paddingTop: '0.5rem'
+  },
+
+  resourceCardArea: {
+    display: 'flex',
+    flexWrap: 'wrap'
+  },
+
+  infoIcon: {
+    color: '#2691D0',
+    cursor: 'pointer',
+    marginLeft: '0.2rem',
+    height: '16px',
+    width: '16px'
+  },
+  addCard: {
```

Comment:
could any of these styles be moved into `app/components/buttons` to be used commonly for all the add cards on the list pages?

Reply:
I don't think the icon could, because the abstraction used doesn't actually specify a specific action, and the margins and width/height also seem page specific, but I think the font size, weight, and height could be.